horse/models/user.py

```diff
@@ -0,0 +1,20 @@
+class User:
+    def __init__(
+            self, user_id, name, followed_users_ids=None, liked_movies_ids=None
+    ):
+        self.user_id = user_id
+        self.name = name
+        self._followed_users_ids = followed_users_ids or []
+        self._liked_movies_ids = liked_movies_ids or []
+
+    def get_followed_users_ids(self):
+        return list(self._followed_users_ids)
+
+    def add_to_followed_users_ids(self, user_id):
+        self._followed_users_ids.append(user_id)
+
+    def get_liked_movies_ids(self):
+        return list(self._liked_movies_ids)
+
+    def add_to_liked_movies_ids(self, movie_id):
```

Comment:
I'm not sure if we'll need all the `get_*_id` methods - intuition tells me that it will be much simpler to just use plain relations - eg. user has a list of movies that she likes and users that she follows. That way recommendation service will not have to worry about accessing object repositories.